src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -59,6 +59,31 @@ public function __construct(ContainerInterface $container = null, bool $debug =
         $this->sessionOptions = $sessionOptions;
     }
 
+    private function getSessionOptions(): array
+    {
+        return $this->mergePhpSessionOptions($this->sessionOptions);
+    }
+
+    private function mergePhpSessionOptions(array $sessionOptions): array
+    {
+        $mergedSessionOptions = [];
+
+        foreach (session_get_cookie_params() as $key => $value) {
+            $mergedSessionOptions['cookie_'.$key] = $value;
+        }
+
+        foreach ($sessionOptions as $key => $value) {
+            // do the same logic as in the NativeSessionStorage
+            // @see https://github.com/symfony/symfony/blob/v5.4.0/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php#L392-L394
```

Comment:
please remove this link, the previous line is enough

Reply:
done